ui/app/services/path-help.js

```diff
@@ -291,6 +292,11 @@ export default Service.extend({
       }
       newModel.reopenClass({ merged: true });
       owner.unregister(modelName);
+
+      if (newModel.proto().fieldGroups) {
```

Comment:
can you add some comments here explaining how this is used?  I just imagine running into this file later and trying to understand this conditional.